ouroboros-network-framework/src/Ouroboros/Network/Protocol/Handshake/Unversioned.hs

```diff
@@ -8,6 +8,9 @@ module Ouroboros.Network.Protocol.Handshake.Unversioned
   , unversionedHandshakeCodec
   , unversionedProtocolDataCodec
   , unversionedProtocol
+  , DataFlowProtocolData (..)
```

Comment:
TODO: this file belongs in a testing framework so that it isn't accidentally used by real code.

Reply:
https://github.com/input-output-hk/ouroboros-network/issues/3474